src/Microsoft.AspNetCore.DataProtection.AzureStorage/baseline.netcore.json

```diff
@@ -0,0 +1,156 @@
+{
+  "AssemblyIdentity": "Microsoft.AspNetCore.DataProtection.AzureStorage, Version=2.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60",
```

Comment:
The baseline file should be for 1.0.0, not 1.1.2. But, it's correct check the file in here and perhaps in the `rel/1.0.3` or `patch/*` branches.